### ISSUE TITLE
Crash with Mythic Upgrades

### ISSUE THREAD

**Issue by solonovamax** [Globox1997/AdventureZ#161]
### Minecraft Version

1.20.x

### Mod Loader Version

0.14.22

### Mod Version

1.4.18

### Describe the Issue

When running mc with Mythic Upgrades, the game crashes at startup with the following exception:
```
[19:43:32] [main/ERROR]: Mixin apply for mod adventurez failed adventurez.mixins.json:PlayerEntityMixin from mod adventurez -> net.minecraft.class_1657: org.spongepowered.asm.mixin.injection.throwables.InvalidInjectionException InjectionPoint(Shift)[@At("INVOKE")] on net/minecraft/class_1657::damageMixin with priority 1000 cannot inject into net/minecraft/class_1657::method_5643(Lnet/minecraft/class_1282;F)Z merged by net.trique.mythicupgrades.mixin.PlayerEntityMixin with priority 1000 [PREINJECT Applicator Phase -> adventurez.mixins.json:PlayerEntityMixin from mod adventurez -> Prepare Injections ->  -> handler$zdd000$adventurez$damageMixin(Lnet/minecraft/class_1282;FLorg/spongepowered/asm/mixin/injection/callback/CallbackInfoReturnable;)V -> Prepare]
org.spongepowered.asm.mixin.injection.throwables.InvalidInjectionException: InjectionPoint(Shift)[@At("INVOKE")] on net/minecraft/class_1657::damageMixin with priority 1000 cannot inject into net/minecraft/class_1657::method_5643(Lnet/minecraft/class_1282;F)Z merged by net.trique.mythicupgrades.mixin.PlayerEntityMixin with priority 1000 [PREINJECT Applicator Phase -> adventurez.mixins.json:PlayerEntityMixin from mod adventurez -> Prepare Injections ->  -> handler$zdd000$adventurez$damageMixin(Lnet/minecraft/class_1282;FLorg/spongepowered/asm/mixin/injection/callback/CallbackInfoReturnable;)V -> Prepare]
```

The issue is caused by their `PlayerEntityMixin`:
https://github.com/TriQue06/mythic-upgrades-1.20.x/blob/409586947cecbd365330a18252b8f3f87e4faeba/src/main/java/net/trique/mythicupgrades/mixin/PlayerEntityMixin.java#L35-L136

See TriQue06/mythic-upgrades-1.20.x#45.

### latest.log

[2023-10-15-6.log](https://github.com/Globox1997/AdventureZ/files/12919090/2023-10-15-6.log)

### Do you use any other mods except the required ones?

Mythic Upgrades is required to reproduce.

**Comment by CyberSteve777** [comment]
Fixed in 2.3.2 of Mythic Upgrades, you can test it now

**Comment by Globox1997** [comment]
Thanks for mythic dev.
